lucuma-schemas/src/test/scala/lucuma/schemas/decoders/InputStreamSuite.scala

```diff
@@ -14,22 +15,12 @@ import java.io.FileInputStream
 import java.nio.file.Paths
 
 trait InputStreamSuite extends CatsEffectSuite {
-  def inputStream(f: File): Resource[IO, FileInputStream] =
-    Resource.make {
-      IO.blocking(new FileInputStream(f)) // build
-    } { inStream =>
-      IO.blocking(inStream.close()).handleErrorWith(_ => IO.unit) // release
-    }
 
   def jsonResult(jsonFile: String): IO[Json] = {
-    val url  = getClass.getResource(jsonFile)
-    val file = Paths.get(url.toURI).toFile
-
-    inputStream(file).use { inStream =>
-      for {
-        str  <- IO.blocking(scala.io.Source.fromInputStream(inStream).mkString)
-        json <- IO.fromEither(parse(str))
-      } yield json
+    val path = Path(s"lucuma-schemas/src/test/resources") / jsonFile
```

Comment:
Is there a way to avoid the hardcoded path?

Reply:
Scala.js doesn't know about resources. We can add the `BuildInfo` plugin and have the build generate sources with the path.

Reply:
Or we can revert my changes here and just not run the tests on JS. They were not being run on JS before, maybe it's not that important.

Reply:
It's probably a good idea to run the tests in JS. Let's leave it as it is, having the hardcoded path is OK in this case.